wrappers/services.go

```diff
@@ -705,6 +760,48 @@ func (es *ensureSnapServicesContext) ensureSnapSlices(quotaGroups *quota.QuotaGr
 	return nil
 }
 
+func (es *ensureSnapServicesContext) ensureSnapJournaldUnits(quotaGroups *quota.QuotaGroupSet) error {
+	handleJournalModification := func(grp *quota.Group, path string, content []byte) error {
+		old, modifiedFile, err := tryFileUpdate(path, content)
+		if err != nil {
+			return err
+		}
+
+		if modifiedFile {
```

Comment:
Is this tested, i.e. that no change means no restart (probably implicit somewhere, mostly wondering)

Reply:
This would be caught by the unit tests - they keep track of all the systemd calls/system calls, and if one extra is called or one goes missing half of all unit tests in services_tests.go go failing. But it's definitely not visible from these changes so I'll leave this comment open